### ISSUE TITLE
Right click - zooming + context menu

### ISSUE THREAD

**Issue by Cu8eeeR** [fancyapps/fancybox#1354]
There is a small bug:
1) when the picture is resized (zoomed out, big resolution) and you click on it with the right mouse button -> image is zoomed in and the context menu is opened (I think it should NOT be zoomed in, only the context menu should be shown)

2) when the image is not resized (it is small) -> right click -> fancybox is closed automatically, no way how to open right context menu

**Comment by fancyapps** [comment]
Are you sure you are using the latest version?

**Comment by Cu8eeeR** [comment]
I use version which is about 7 days old... anyway I will check it with your latest GitHub version.

Anyway, do you have any ETA for v3.1? I use custom css modifications so it is quite difficult to track all changes in css between new versions . It would be better to make it only once so if you need someone for testing v3.1, let me know, Thank you!

**Comment by fancyapps** [comment]
I remember that this was fixed about a month ago.

It is hard to give ETA, I have managed to implement most requested features (like custom interaction) except transitions. You see, it is tricky to implement touch interactions and transition effects simultaneously.  I have tried different techniques, but they all had some drawbacks and now I got a new idea how to implement this, but I have to rewrite large pieces of code.
